### PR TITLE
273 opening the page no example schema but show a dialog what the user wants to do

### PR DESCRIPTION
**Pull request by Keyshal-211**

### CODE REVIEW

**Comment by Keyshal-211** [comment]
all working. except fetchschemaFrom URL

**Comment by Keyshal-211** [comment]
Ready for review.

File: frontend/config-assistant/src/components/CombinedEditorComponent.vue [hunk 1]
```diff
@@ -17,7 +16,10 @@ import {useToast} from 'primevue/usetoast';
 import {useConfirm} from 'primevue/useconfirm';
 import {confirmationService} from '@/helpers/confirmationService';
 import {toastService} from '@/helpers/toastService';
-
+import RadioButton from 'primevue/radiobutton';
+import Dialog from 'primevue/dialog';
```
Comment (PaulBredl) [inline frontend/config-assistant/src/components/CombinedEditorComponent.vue:20]: are all those imports necessary?

File: frontend/config-assistant/src/components/toolbar/TopToolbar.vue [hunk 2]
```diff
@@ -285,9 +324,21 @@ watchDebounced(
 </script>
 
 <template>
-  <Dialog v-model:visible="topMenuBar.showDialog.value">
+  <Dialog v-model:visible="showDialog" header="Select a Schema">
```
Comment (PaulBredl) [inline frontend/config-assistant/src/components/toolbar/TopToolbar.vue:327]: Can you please make a new component "InitialSchemaSelectionDialog" out of this? 
The top toolbar file is already way to big and hard to understand. You can use the emits to emit a selection event, depending on which the corresponding functionality is called. you can also defineExpose() to have a "show" method to be called from outside (see the SchemInfoOverlayPanel, there I used this)

**Review by PaulBredl — CHANGES_REQUESTED** [review]
Functionally correct, though code could be more clean be creating a new component for the dialog (we should do for other dialogs too in the future)

File: frontend/config-assistant/src/components/toolbar/TopToolbar.vue (hunk 2 above)
Comment (Logende) [reply frontend/config-assistant/src/components/toolbar/TopToolbar.vue:327]: If you add a new component for this feature, a good place to import and use this component would be inside the FileEditorView.vue or CombinedEditorComponent.vue.

File: frontend/config-assistant/src/components/CombinedEditorComponent.vue (hunk 1 above)
Comment (Keyshal-211) [reply frontend/config-assistant/src/components/CombinedEditorComponent.vue:20]: yes it is necessary.

File: frontend/config-assistant/src/components/toolbar/TopToolbar.vue [hunk 3]
```diff
@@ -243,6 +267,12 @@ watch(storeToRefs(useSessionStore()).fileData, () => {
 const searchTerm: Ref<string> = ref('');
 const searchBarVisible = ref(false);
 
+const initialSchemaSelectionDialog = ref();
+// Function to show the category selection dialog
+const showCategoryDialog = () => {
```
Comment (PaulBredl) [inline frontend/config-assistant/src/components/toolbar/TopToolbar.vue:272]: I would only quickly update the name of this function to `showInitialSchemaDialog` then everything is very nice

File: frontend/config-assistant/src/components/CombinedEditorComponent.vue (hunk 1 above)
Comment (PaulBredl) [reply frontend/config-assistant/src/components/CombinedEditorComponent.vue:20]: I checked out the branch for me there all greyed out: 
image

Comment (Keyshal-211) [reply frontend/config-assistant/src/components/CombinedEditorComponent.vue:20]: its not anymore now @PaulBredl

**Review by PaulBredl — APPROVED** [review]

**Review by Logende — APPROVED** [review]

**Pull request merged**